src/website/src/pages/ec/components/index.md

```diff
@@ -3,4 +3,6 @@ title: Components
 order: 6
 ---
 
-(work in progress)
```

Comment:
I'm just unsure whether it's already time to remove this label "work in progress", because it's still the case, at least in a few of these "sections" :)

Reply:
It will always be "work in progress" 😄